resources/css/app.css

```diff
@@ -1,18 +1,138 @@
-@import 'components/vue-slider';
-@import 'components/price-slider';
-@import 'components/pagination';
-@import 'components/autocomplete';
+@import './components/vue-slider.css' layer(base);
+@import './components/price-slider.css' layer(base);
+@import './components/pagination.css' layer(base);
+@import './components/autocomplete.css' layer(base);
 
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
+@import 'tailwindcss';
+@config "../../tailwind.config.js";
+
+@plugin '@tailwindcss/forms';
+@plugin '@tailwindcss/typography';
+@plugin 'tailwind-scrollbar-hide';
+
+@theme {
+    --color-primary: color-mix(in srgb, var(--primary, #2fbc85) calc(100% * 1), transparent);
+    --color-primary-text: color-mix(in srgb, var(--primary-text, #fff) calc(100% * 1), transparent);
+    --color-secondary: color-mix(in srgb, var(--secondary, #202f60) calc(100% * 1), transparent);
+    --color-secondary-text: color-mix(in srgb, var(--secondary-text, #fff) calc(100% * 1), transparent);
+    --color-conversion: color-mix(in srgb, var(--conversion, #22c55e) calc(100% * 1), transparent);
+    --color-conversion-text: color-mix(in srgb, var(--conversion-text, #fff) calc(100% * 1), transparent);
+    --color-foreground-emphasis: color-mix(in srgb, var(--foreground-emphasis, #0f172a) calc(100% * 1), transparent);
+    --color-foreground: color-mix(in srgb, var(--foreground, #1e293b) calc(100% * 1), transparent);
```

Comment:
As per: https://tailwindcss.com/docs/colors#referencing-in-css and previously within the `tailwind.config.js` we can reference existing colors;

```
--color-foreground: color-mix(in srgb, var(--foreground, var(--color-slate-800)) calc(100% * 1), transparent);
```

Demo: https://play.tailwindcss.com/QB164GxvSS